### PR TITLE
media_gallery.js documentation

### LINKED ISSUE #12: Document wp-admin/js/media-gallery.js

### PR DESCRIPTION
**Pull request by moorscode**
This file has been deprecated since 3.5.0

Fixes #12

### CODE REVIEW

File: src/wp-admin/js/media-gallery.js [hunk 1]
```diff
@@ -1,15 +1,28 @@
 /* global ajaxurl */
+
+/**
+ * This file is used on media-upload.php which has been replaced by media-new.php and upload.php
+ * Deprecated since 3.5.0
+ */
 jQuery(function($){
+	/**
+	 * Adds an click event handler to the element with a 'wp-gallery' class.
```
Comment (terw-dan) [inline src/wp-admin/js/media-gallery.js]: should be 'a click' in stead of 'an click'

File: src/wp-admin/js/media-gallery.js [hunk 2]
```diff
@@ -1,15 +1,28 @@
...
+	 */
 	$( 'body' ).bind( 'click.wp-gallery', function(e){
 		var target = $( e.target ), id, img_size;
 
 		if ( target.hasClass( 'wp-set-header' ) ) {
+			// Opens the image to preview it full size.
 			( window.dialogArguments || opener || parent || top ).location.href = target.data( 'location' );
 			e.preventDefault();
 		} else if ( target.hasClass( 'wp-set-background' ) ) {
+			// Set the image as background of the theme.
```
Comment (terw-dan) [inline src/wp-admin/js/media-gallery.js]: should be 'set' in stead of 'sets'

**Comment by terw-dan** [comment]
CR done

**Comment by moorscode** [comment]
🎉  Ticket created: https://core.trac.wordpress.org/ticket/37717

**Pull request closed without merging**